 i686-mingw32-gcc -mwindows -mms-bitfields -c -DLOCALEDIR=\"/usr/share/locale\" -DLOCALE_ALIAS_PATH=\"/usr/share/locale\" -DLIBDIR=\"/usr/lib\" -DBUILDING_LIBINTL -DBUILDING_DLL -DIN_LIBINTL -DENABLE_RELOCATABLE=1 -DIN_LIBRARY -DINSTALLDIR=\"/usr/lib\" -DNO_XMALLOC -Dset_relocation_prefix=libintl_set_relocation_prefix -Drelocate=libintl_relocate -DDEPENDS_ON_LIBICONV=1 -DHAVE_CONFIG_H -I. -I/home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl -I.. -g -O2 -fvisibility=hidden /home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl/log.c  -DDLL_EXPORT -DPIC -o .libs/log.o
/home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl/log.c: In functie ‘_nl_log_untranslated’:
/home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl/log.c:116: let op: visibility attribute not supported in this configuration; ignored
/home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl/localename.c: In functie ‘_nl_locale_name_default’:
/home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl/localename.c:1145: fout: herhaalde case-waarde
/home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl/localename.c:1144: fout: eerder hier gebruikt
/home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl/localename.c:1359: fout: herhaalde case-waarde
/home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl/localename.c:1358: fout: eerder hier gebruikt
/home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl/localename.c:1367: fout: herhaalde case-waarde
/home/janneke/vc/gub/target/mingw/src/gettext-0.15/gettext-runtime/intl/localename.c:1366: fout: eerder hier gebruikt
make[3]: *** [localename.lo] Fout 1
make[3]: *** Wachten op onvoltooide taken...

--- gettext-0.15/gettext-runtime/intl/localename.c.orig	2008-10-24 16:56:13.000000000 +0200
+++ gettext-0.15/gettext-runtime/intl/localename.c	2008-10-24 16:58:44.000000000 +0200
@@ -1142,7 +1142,6 @@ _nl_locale_name_default (void)
 	switch (sub)
 	  {
 	  case SUBLANG_BENGALI_INDIA: return "bn_IN";
-	  case SUBLANG_BENGALI_BANGLADESH: return "bn_BD";
 	  }
 	return "bn";
       case LANG_BULGARIAN: return "bg_BG";
@@ -1356,7 +1355,6 @@ _nl_locale_name_default (void)
 	switch (sub)
 	  {
 	  case SUBLANG_PUNJABI_INDIA: return "pa_IN"; /* Gurmukhi script */
-	  case SUBLANG_PUNJABI_PAKISTAN: return "pa_PK"; /* Arabic script */
 	  }
 	return "pa";
       case LANG_RHAETO_ROMANCE: return "rm_CH";
@@ -1364,7 +1362,6 @@ _nl_locale_name_default (void)
 	switch (sub)
 	  {
 	  case SUBLANG_ROMANIAN_ROMANIA: return "ro_RO";
-	  case SUBLANG_ROMANIAN_MOLDOVA: return "ro_MD";
 	  }
 	return "ro";
       case LANG_RUSSIAN:
